sdk/ai/azure-ai-projects/samples/agents/sample_agents_with_code_interpreter_file_attachment.py

```diff
@@ -85,7 +85,7 @@
     messages = project_client.agents.get_messages(thread_id=thread.id)
     print(f"Messages: {messages}")
 
-    last_msg = messages.get_last_text_message_by_sender("assistant")
+    last_msg = messages.get_last_text_message_by_role(MessageRole.AGENT)
```

Comment:
was this only place?

Reply:
yes and one in async. i could not find others.